.mergify.yml

```diff
@@ -0,0 +1,9 @@
+---
+pull_request_rules:
+  - name: backport
+    conditions:
+      - label=stable backport potential
```

Comment:
For some reason the label we use is `backport stable potential` instead

Reply:
Heh, ok everything else uses `stable backport potential` (I just copied this from terra), but not a big deal updated it in: https://github.com/Qiskit/qiskit-experiments/pull/1064/commits/14934081d1d9cc65b3f9b899ec7fcb8914dddc32